packages/terra-button/CHANGELOG.md

```diff
@@ -16,7 +18,6 @@ Unreleased
 
 3.22.0 - (September 3, 2019)
 ------------------
-### Changed
```

Comment:
I think this was accidentally removed, we should add it back.

Reply:
Fixed in 7593ab7